### PR TITLE
add default cases for sync mode

### LINKED ISSUE #412: Document fast sync now being the default on named networks
## Describe the bug

The current default syncing mode for Besu is full sync with no pruning. On named networks (see [this list|https://besu.hyperledger.org/en/latest/Reference/CLI/CLI-Syntax/#network], excluding dev), the default should be changed to fast sync and pruning.

- [ ] Missing content
- [x] Outdated content
- [ ] Wrong content
- [ ] Confusing or misleading content
- [ ] Other

## The broken page

https://besu.hyperledger.org/en/latest/Concepts/Pruning/#fast-synchronization and any place where fast sync is mentionned should be checked

## Change suggestion

On named networks the default should be changed to fast sync and pruning

## Screenshots

na

## More context

Document changes from hyperledger/besu#498

### PR DESCRIPTION
**Pull request by NicolasMassart**
Add warning in the concept page that sync mode is now defaulting to fast for named networks except the development one.
Add a new column in named networks table make the default sync mode for each network more visible.

fixes #412

### CODE REVIEW

File: docs/Concepts/Pruning.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ which starts from the genesis block and reprocesses all transactions, fast synch
 ([`--sync-mode=FAST`](../Reference/CLI/CLI-Syntax.md#sync-mode)) downloads the block headers and
 transaction receipts, and verifies the chain of block headers from the genesis block.
 
+!!!important
+    Fast synchronization is the default setting for named networks selected when starting Besu with
+    the [`--network` CLI option](../Reference/CLI/CLI-Syntax.md#network), except for `dev` development network..
```
Comment (bgravenorst) [inline docs/Concepts/Pruning.md:43]: Extra fullstop (.) should be removed.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -2139,12 +2139,16 @@ BESU_SYNC_MODE=FAST
 sync-mode="FAST"
 ```
 
-The synchronization mode. The options are `FAST` and `FULL`. The default is `FULL`.
+The synchronization mode. The options are `FAST` and `FULL`.
+
+* The default is `FULL` when not using the [`--network`](#network) option.
```
Comment (RatanRSur) [inline docs/Reference/CLI/CLI-Syntax.md:2144]: Technically you can get a FAST sync even when not using the network option. If you don't supply anything (genesis or network) then it's mainnet, so it's fast.

File: docs/Concepts/Pruning.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Concepts/Pruning.md]: I've pushed this edit

File: docs/Reference/CLI/CLI-Syntax.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md:2144]: @RatanRSur - so the default is FULL if you don't name a network and specify a genesis file?

Comment (RatanRSur) [reply docs/Reference/CLI/CLI-Syntax.md:2144]: Yes

Comment (MadelineMurray) [reply docs/Reference/CLI/CLI-Syntax.md]: Updated

**Pull request merged**